### PR TITLE
fix: instantiate cookiesservice in logincontroller

### PR DESCRIPTION
**Pull request by nichonien**
This PR instantiates `CookiesService` in `LoginController` and removes it from constructor params.

### CODE REVIEW

**Review by JGiter — COMMENTED** [review]
@nichonien could you point out what is the benefirs of manual instantiating of `CookiesService`?

**Comment by nichonien** [comment]
> @nichonien could you point out what is the benefirs of manual instantiating of `CookiesService`?

@JGiter There is no actual benefit, but honestly this instance coming from outside doesn't add any value or provide anything new with comparison to an internal instantiation (but eases the implementation by reducing constructor params). Secondly, this is an attempt to check if the `Snyk` code scan can pick the `security` property in `cookies` if it is created within the same class.

File: src/modules/auth/login.controller.ts [hunk 1]
```diff
@@ -20,9 +20,9 @@ import { RoleService } from '../role/role.service';
 @ApiTags('Auth')
 @Controller({ version: '1' })
 export class LoginController {
+  private cookiesServices = new CookiesServices();
```
Comment (jrhender) [inline src/modules/auth/login.controller.ts:23]: I think this is unlikely to change snyk's results and we lose the benefits of nest Dependency Injection. I think that I would try copying the `CookiesService` code directly into the LoginController

**Review by jrhender — COMMENTED** [review]
I'm at bit skeptical that this will work, but you never know.

Could you try running the Snyk cli against this branch to see if we can test it before merging? https://docs.snyk.io/snyk-cli/test-for-vulnerabilities/grouping-projects-by-branch-or-version

**Comment by jrhender** [comment]
@nichonien shall we close this PR?

**Comment by nichonien** [comment]
Closing as this is not needed anymore.

**Pull request closed without merging**